src/sentry/utils/email.py

```diff
@@ -147,6 +153,22 @@ def get_from_email_domain():
     return _from_email_domain_cache[1]
 
 
+def create_fake_email(unique_id, namespace):
+    """
+    Generate a fake email of the form: {unique_id}@{namespace}.{FAKE_EMAIL_DOMAIN}
+
+    For example: c74e5b75-e037-4e75-ad27-1a0d21a6b203@cloudfoundry.sentry-fake
+    """
+    return "{}@{}.{}".format(unique_id, namespace, FAKE_EMAIL_DOMAIN)
+
+
+def is_fake_email(email):
+    """
+    Returns True if the provided email matches the fake email pattern.
+    """
+    return bool(FAKE_EMAIL_RE.search(email))
```

Comment:
Since this is just an ending TLD, can't we just change this to `email.endswith('.sentry-fake')` ?